### DISCUSSION TITLE
Honkai: Star Rail support?

### DISCUSSION THREAD

**Discussion by MajorTazer** [vermaysha/hoyolab-api#115] [Ideas]
With the recent release of Honkai: Star Rail, is it possible to add support for this game? Automating check-ins make life a lot easier. If I can help by getting endpoints, etc. please let me know!

**Comment by vermaysha** [comment]
Yes, it is very possible, it will be added soon.

**Reply by MajorTazer** [reply]
Perfect, thank you!
